app/code/Magento/Bundle/view/adminhtml/templates/product/composite/fieldset/options/type/checkbox.phtml

```diff
@@ -59,3 +59,13 @@
         </div>
     </div>
 </div>
+<fieldset id="product_composite_configure_fields_qty"
+          class="fieldset product-composite-qty-block admin__fieldset <?= $block->getIsLastFieldset() ? 'last-fieldset' : '' ?>">
+    <div class="field admin__field">
+        <label class="label admin__field-label"><span><?= $block->escapeHtml(__('Quantity')) ?></span></label>
+        <div class="control admin__field-control">
+            <input id="product_composite_configure_input_qty" class="input-text admin__control-text qty" type="text" name="qty" value="<?= $block->getQtyValue() * 1 ?>">
```

Comment:
Not really sure why do you use `$block->getQtyValue() * 1`
What's wrong with `(int) $block->getQtyValue()`. Does it have some problems with static tests?